kf_hungarian_tracker/package.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>kf_hungarian_tracker</name>
+  <version>0.0.1</version>
+  <description>Use Kalman Filter and Hungarian algorithm to track multiple objects</description>
+  <maintainer email="stevenmacenski@gmail.com">Steven Macenski</maintainer>
+  <maintainer email="csj15thu@gmail.com">Shengjian Chen</maintainer>
+  <license>Apache-2.0</license>
+
+  <test_depend>ament_copyright</test_depend>
+  <test_depend>ament_flake8</test_depend>
+  <test_depend>ament_pep257</test_depend>
+  <test_depend>python3-pytest</test_depend>
+
+  <exec_depend>launch_ros</exec_depend>
+
+  <exec_depend>numpy</exec_depend>
+  <exec_depend>scipy</exec_depend>
+  <exec_depend>nav2_dynamic_msgs</exec_depend>
+  <exec_depend>visualization_msgs</exec_depend>
```

Comment:
tf2_ros

Reply:
Take a look at all of your imports and make sure they're represented here unless they come with python3 default install on all platforms.